### PR TITLE
Added Documentation

### PR DESCRIPTION
**Pull request by codguy**
- Added Documentation
- Update css for buttons

### CODE REVIEW

File: popup.css [hunk 1]
```diff
@@ -15,7 +15,21 @@ body {
...
+    margin: 3px;
+    font-size: 15px;
+    font-family: 'Courier New', Courier, monospace;
+    background-color: orange;
+    border-radius: 1px;
+  }
+  
+    
+  button:hover{
+    padding: 10px 20px;
```
Comment (bobbydevelopers) [inline popup.css:27]: no need to repeat same code

Comment (codguy) [reply popup.css]: Added changes

Comment (bobbydevelopers) [reply popup.css]: add proper documentation.

**Comment by bobbydevelopers** [comment]
done

**Pull request merged**